src/main/kotlin/utils/CoverageFileStatistics.kt

```diff
@@ -0,0 +1,64 @@
+package utils
+import org.xml.sax.Attributes
+import org.xml.sax.InputSource
+import org.xml.sax.helpers.DefaultHandler
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import javax.xml.parsers.SAXParserFactory
+import kotlin.math.min
+
+object CoverageFileStatistics {
```

Comment:
please add a class comment

Reply:
please add a unit test for this class (e.g. by having it print to a printstream instead of directly to stdout and verifying the output in the unit test)